### PR TITLE
Travis-ified application

### PR DESCRIPTION
**Pull request by kfriedman**

### CODE REVIEW

**Review by holingpoon — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -45,25 +49,26 @@ Various files are used to configure and deploy the Lambda.
 
 ### package.json
 
-Configures `npm run` deployment commands for each environment and sets the proper AWS Lambda VPC and
-security group.
+Configures `npm run` commands for each environment for deployment and testing. Deployment commands may also sets
```
Comment (gkallenberg) [inline README.md]: Wording here should be singular 'set' instead of 'sets.'

File: composer.json [hunk 2]
```diff
@@ -1,10 +1,13 @@
 {
-  "name": "nypl-services",
-  "description": "NYPL Services Package",
+  "name": "nypl-bibService",
```
Comment (gkallenberg) [inline composer.json]: Mix of hyphens and camel case? Not 'nypl-bib-service' ??

**Comment by gkallenberg** [comment]
All in all, looks good. Just a few nit-picks in the code comments.

**Comment by kfriedman** [comment]
Thank you for the nits. Made the changes you recommended.

**Pull request merged**